Safety/Enter-Observatory-Spaces/Enter-AuxTel-Telescope.rst

```diff
@@ -0,0 +1,190 @@
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Gonzalo Aravena*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Alysha Shugart*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Safe-entry-to-AuxTel:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+####################
+Safe entry to AuxTel
+####################
+
+.. _Safe-entry-to-AuxTel-Overview:
+
+Overview
+========
+
+This document aims to provide the necessary guidelines and procedures for safely entering AuxTel. 
+It covers aspects related to access during nighttime operations, ventilation, calibrations, and daytime access. 
+Ensure you follow these instructions to guarantee your safety and the safety of the equipment.
+
+
+Safety equipment for all entries
+================================
+
+Each entry to the telescope, regardless of the reason, must mandatorily use the following safety equipment:
+
+- Helmet
+
+- Reflective vest
+
+- Safety shoes
+
+.. figure:: ./_static/Notice.png
+    :name: notice
+    :scale: 25 %
+
+    Safety Equipment
+
+.. note:: 
+   
+   If you do not have any of these safety items, please contact the safety staff at the summit. 
+
+
+
+.. _Safe-entry-to-AuxTel-access-during-daytime:
+
+
+Access During Daytime
+=====================
+
+
+1. **Conditional access**
+
+   - Ensure that no images are being taken and no work is being done in the AT dome that might be disrupted upon entering the telescope.
+
+
+2. **Entry door key**
+
+   - The key to access the main entrance door to the telescope will be hanging on the right side of the door just below the fire extintor sign. It is strictly requested that the keys must be always left in the same place when leaving the telescope.
+
+
+   .. figure:: ./_static/keys.png
+       :name: keys
+       :scale: 25 %
+
+       Key hook
+
+
+
+   .. note:: 
+       Exceptionally, the entrance door will currently be left unlocked.
+
+
+
+3. **Communication Protocols**
```

Comment:
I think step #3 (communication protocol) should become either Step #1 or Step #2.

Reply:
Agreed. For better readability and logic, I suggest placing it at the beginning as step 1.